pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/TableConfigsRestletResource.java

```diff
@@ -147,16 +150,18 @@ public String getConfig(
   @POST
   @Produces(MediaType.APPLICATION_JSON)
   @Path("/tableConfigs")
-  @ApiOperation(value = "Add the TableConfigs using the tableConfigsStr json",
-      notes = "Add the TableConfigs using the tableConfigsStr json")
-  public SuccessResponse addConfig(
-      String tableConfigsStr,
+  @ApiOperation(value = "Add the TableConfigs using the tableConfigsStr json", notes = "Add the TableConfigs using "
```

Comment:
(code style) Same here, don't change this format because it doesn't improve readability

Reply:
ACK. Intellij auto format is messing it up.